Bottom: 0968550d674366a7444feac2c4f99413334f8695
Top:    083fe104693e36085eaeff25e1c4d2628b62582f
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-11 16:08:43 +0000

Refresh of try-not-using-no-holes-dicer-f

---

diff --git a/src/draw.c b/src/draw.c
index 38abe43..fdb33f7 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -2171,7 +2171,6 @@ DrawPlainPolygon (LayerTypePtr Layer, PolygonTypePtr Polygon)
     }
   else if (Polygon->Clipped)
     {
-#if 0
       if (!Polygon->NoHolesValid)
         {
           ComputeNoHoles (Polygon);
@@ -2185,15 +2184,6 @@ DrawPlainPolygon (LayerTypePtr Layer, PolygonTypePtr Polygon)
             poly.Clipped = poly.Clipped->f;
           } while (poly.Clipped != Polygon->NoHoles);
         }
-#endif
-        {
-          PolygonType poly = *Polygon;
-          poly.Clipped = Polygon->Clipped;
-          do {
-            DrawPolygonLowLevel (&poly, NULL);
-            poly.Clipped = poly.Clipped->f;
-          } while (poly.Clipped != Polygon->Clipped);
-        }
       /* draw other parts of the polygon if fullpoly flag is set */
       /* NB: No "NoHoles" cache for these */
       if (TEST_FLAG (FULLPOLYFLAG, Polygon))
